### PR TITLE
Added changing password function

### PR DESCRIPTION
**Pull request by deswra**

### CODE REVIEW

File: frontend/src/components/ChangePassword/ChangePassword.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            <Card className={`${styles.authFormContainer} shadow-lg mb-5`}>
+                <Card.Body>
+                    {alert.show ? (
+                        <AppAlert
+                            variant={alert.variant}
+                            content={alert.content}
+                            dismissible={alert.dismissible}
+                            setShow={setAlert}
+                        />
+                    ) : null}
```
Comment (sonns) [inline frontend/src/components/ChangePassword/ChangePassword.jsx:63]: @deswra 
You can use Short-circuit operator for this

File: frontend/src/components/ChangePassword/ChangePassword.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        } else {
+            setAlert({
+                content: 'Password successfully changed.',
+                variant: 'success',
+                dismissible: true,
+                show: true,
+            });
+        }
+    };
+
```
[the remark this answers is not in the document]
Comment (deswra) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:40]: It is much cleaner. I will replace the original with it. Thank you.

File: frontend/src/components/ChangePassword/ChangePassword.jsx (hunk 1 above)
Comment (deswra) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:63]: Thanks for letting me know about it, I'll try to use it in the future.

Comment (sonns) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:63]: Short-circuit operator (&&)
for example : 
return (
    <div>
        { showHeader && <Header /> }
    </div>
);

Comment (sonns) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:63]: {alert.show && (
                        <AppAlert
                            variant={alert.variant}
                            content={alert.content}
                            dismissible={alert.dismissible}
                            setShow={setAlert}
                        />
                    )}

Comment (sonns) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:63]: {alert.show ? (
                        \<AppAlert
                            variant={alert.variant}
                            content={alert.content}
                            dismissible={alert.dismissible}
                            setShow={setAlert}
                        \/\>
                    )}

Comment (deswra) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:63]: This was changed in 866e0d7030bd0f1f613319de95dc9d2f9ef07c6e

File: frontend/src/components/ChangePassword/ChangePassword.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (GinRex) [reply frontend/src/components/ChangePassword/ChangePassword.jsx:40]: `content: result.errors ? result.errors : 'Password successfully changed.',
`
is good but we still have a duplication, how about:
`content: result.errors ?? 'Password successfully changed.',
`

**Pull request merged**